apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -61,6 +63,8 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     enum VoterState { Absent, Yea, Nay }
 
     enum VoteStatus {
+        // Note: may be able to find a better name than "active". This state just includes all votes
+        // that have not been challenged or executed (thus including votes that have ended).
```

Comment:
I'm having a hard time finding a better word to describe this state, so perhaps updating the "ongoing vote" comment would be easier.

"Active" is a bit loose but actually describes this state pretty well. Other choices I didn't like very much: "Normal", "Not disrupted", "Base", "Basic" (keeping in mind that ruling as accepted brings the vote back to "Active").

Reply:
I agree but I don't like much the other options either haha.. let's go with "Normal" for now tho. Addressed in 9a29ff9238860352b9bbd6e1c71ac08871eb194d